index.js

```diff
@@ -0,0 +1,59 @@
+import { Book, BookStore, showBooks } from './modules/books.js';
+import { DateTime } from './modules/luxon.js';
+
+const bookListDiv = document.getElementById('table');
+const formDiv = document.getElementById('form');
+const contactDiv = document.getElementById('contact');
+const contactButton = document.getElementById('li-contact');
+const liForm = document.getElementById('li-list');
+const listAdd = document.getElementById('li-add');
+
+function constructListPage() {
+  bookListDiv.style.display = 'block';
+  formDiv.style.display = 'none';
+  contactDiv.style.display = 'none';
+}
+
+function constructFormPage() {
+  bookListDiv.style.display = 'none';
+  formDiv.style.display = 'block';
+  contactDiv.style.display = 'none';
+}
+
+function constructContactPage() {
```

Comment:
- [ ] As this is ES6 it is good practice to use arrow functions 

![image](https://user-images.githubusercontent.com/75126481/165133076-9287c4c1-0a08-4f30-afe3-980f4ccd3a54.png)